docs/DataProviders.md

```diff
@@ -467,8 +467,8 @@ Request Type         | Response format
 `CREATE`             | `{ data: {Record} }`
 `UPDATE`             | `{ data: {Record} }`
 `UPDATE_MANY`        | `{ data: {mixed[]} }` The ids which have been updated
-`DELETE`             | `{ data: {Record} }`
-`DELETE_MANY`        | `{ data: {mixed[]} }` The ids which have been deleted
+`DELETE`             | `{ data: {Record} }` The resource that had been deleted or nothing
```

Comment:
```suggestion
`DELETE`             | `{ data: {Record} }` The record that has been deleted
```

Reply:
I would precise, to the data provider maintainers that this data isn't required.